### PR TITLE
Add option to disable trackers preview from the iframe directly

### LINKED ISSUE #4: Opt-out button for Trackers Preview
On the Trackers Preview we want to have a button to disable this feature. It should send a message to extension background - the background will decide how to implement the settings.

Visually, the button itself should be added to Trackers Preview:
Screenshot 2022-01-26 at 17 21 48

and can look like links in the footer of Ghostery 9 popup:
Screenshot 2022-01-26 at 17 22 31

It should say "Disable Trackers Preview" - the text should come from translations.

Additionally we should replace text "Trackers on this site" with "Trackers Preview", it should also come from translations.

### PR DESCRIPTION
**Pull request by smalluban**
Fixes #4 .

### CODE REVIEW

File: packages/serp-report/src/pages/iframe/wtm-report.js [hunk 1]
```diff
@@ -36,121 +36,159 @@ function requestClose() {
...
-    )}
+  confirmDisabled: false,
+  render: ({ confirmDisabled }) => html`
+    <panel-header domain=${domain}>
+      <button class="svg-button" onclick="${requestClose}">${close}</button>
+    </panel-header>
 
     <main>
-      <h1>${t('android_site_blocking_header')}</h1>
+      <h1>${t('wtm_trackers_preview_title')}</h1>
```
Comment (philipp-classen) [inline packages/serp-report/src/pages/iframe/wtm-report.js:52]: That would need to be added as a translation at a difference place, right (ghostery-extension + ghostery-dnr-extension)?

File: packages/serp-report/src/content_scripts/serp-report.js [hunk 2]
```diff
@@ -168,6 +177,13 @@ function renderWheel(WTMTrackerWheel, anchor, stats) {
       return;
     }
 
+    if (message.data === 'WTMReportDisable') {
+      closePopups();
+      elements.forEach(removeWheel);
+      chrome.runtime.sendMessage({ action: 'disableWTMReport' });
```
Comment (chrmod) [inline packages/serp-report/src/content_scripts/serp-report.js:183]: Would be good to define actions as constants and reuse in all contexts but the fact that content script don't support JavaScript modules makes it awkward.

File: packages/serp-report/src/pages/iframe/wtm-report.js (hunk 1 above)
Comment (chrmod) [reply packages/serp-report/src/pages/iframe/wtm-report.js:52]: yes, we sync translation files with transifex

**Review by chrmod — APPROVED** [review]

File: packages/serp-report/src/content_scripts/serp-report.js (hunk 2 above)
Comment (smalluban) [reply packages/serp-report/src/content_scripts/serp-report.js:183]: I thought about it, but as it requires to use dynamic import, I didn't want to make to complex code here.

File: packages/serp-report/src/pages/iframe/wtm-report.js (hunk 1 above)
Comment (smalluban) [reply packages/serp-report/src/pages/iframe/wtm-report.js:52]: Unfortunately, we don't have yet process to keep translations here, so they must be provided by the applications. And we have a PR to DNR, that have them.

Comment (chrmod) [reply packages/serp-report/src/pages/iframe/wtm-report.js:52]: actually, for simplicity we could have all transifex translation in this repo as well.

**Pull request merged**